docs/configuration-parameters.md

```diff
@@ -52,6 +52,18 @@ This section provides the details of all the configurations required for the Car
 | carbon.trash.retention.days | 7 | This parameter specifies the number of days after which the timestamp based subdirectories are expired in the trash folder. Allowed Min value = 0, Allowed Max Value = 365 days|
 | carbon.clean.file.force.allowed | false | This parameter specifies if the clean files operation with force option is allowed or not.|
 | carbon.cdc.minmax.pruning.enabled | false | This parameter defines whether the min max pruning to be performed on the target table based on the source data. It will be useful when data is not sparse across target table which results in better pruning.|
+| carbon.storelocation | (none) | This parameter defines the path on DFS where carbondata files and metadata will be stored. |
```

Comment:
carbon.storelocation is need to configure now? does it change to spark.sql.warehouse.dir ?
@QiangCai @kumarvishal09

Reply:
Taken care of.